schedule/yast/sle/flows/default_sle15sp3_x86.yaml

```diff
@@ -15,6 +15,7 @@ registration:
   - installation/registration/register_via_scc
 extension_module_selection:
   - installation/module_registration/register_extensions_and_modules
+  - installation/module_registration/add_ltss_regcode
   - installation/module_registration/add_additional_regcodes
```

Comment:
as we is not added when we have ltss, we don't need `add_additional_regcodes` probably.